compute.py

```diff
@@ -0,0 +1,36 @@
+from const import packets_str, size_str, dts_time_str, duration_time_str, x_str, y_str
+from ffmpeg import get_packets_info, get_qp_data
+from graph import generate_line_graph
+
+
+# data computation
```

Comment:
This comment doesnt add anything. The function docstring comment is fine.

Reply:
removed